source/heroes/ControllableHero.hx

```diff
@@ -0,0 +1,38 @@
+package heroes;
+
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.FlxG;
+
+class ControllableHero extends FlxSprite {
+    var DirectionX:Float;
+    var DirectionY:Float;
+
+    public function new(X:Float=0, Y:Float=0)
+    {
+        super(X, Y);
+        makeGraphic(25, 50, 0xFF006118);
```

Comment:
It's good practice to take numbers like these and store them in class-level variables - static properties that can't be changed is usually a good policy for const-type stuff. 

Then we can give them names and make the intent of our code clearer - the player's `starting_width` should be `25`, the player's `starting_height` should be `50`, and their `starting_color` should be `0xFF006118`

Let's implement this sort of fix here and elsewhere in the project.